archive/k/kotlin/fibonacci.kt

```diff
@@ -0,0 +1,25 @@
+import kotlin.system.exitProcess
+
+// First arg is number of iterations to run
+fun main(args: Array<String>) {
+    if (args.size != 1 && args[0].toIntOrNull() == null)
+        exitProcess(1)
+
+    val iterations = args[0].toInt()
+
+    var j: Int
+    var k = 0
+    var l = 1
+
+    for (i in 1..iterations) {
+
+        print(l)
+
+        if (i != iterations) {
+            print(", ")
```

Comment:
I'm not 100% positive, but I think the usual solution prints each number on a new line.

Reply:
I think you're right, I quickly glanced back [here](https://github.com/TheRenegadeCoder/sample-programs/blob/master/CONTRIBUTING.md#fibonacci-sequence-rules) and looked at the first code block rather than second (which is the desired format).